language/fast_text.py

```diff
@@ -0,0 +1,45 @@
+try:
+    import fasttext
+except:
+    import sys
+    import subprocess
+    print("Installing fast_text library")
+    subprocess.check_call([sys.executable, '-m', 'pip', 'install','fasttext'])
```

Comment:
Hi @noviluni,
Can I use this kind of in program installation of the additional libraries?

Reply:
IMO we should not try to install anything automatically, this is on the user to decide.

Reply:
Hi @gavishpoddar, no you shouldn't :slightly_smiling_face: 

I could try to explain the reasons behind it (is not pythonic, you are installing something without asking the user, etc), but I think it's better to talk about possible approaches.

Probably the best approach is adding an optional dependency. These can be indicated in the `setup.py` file with the `extras_require` property. We do it to be able to use the `hijri` and `jalali` calendars. To use them you have to do: `pip install dateparser[calendars]`. 

I think you should find some examples on the internet (apart from the `calendars` example in this project). Let me know if you don't know how to do it.

(What you could do is capturing the `ImportError` and give a hint on how to enable it.)

Reply:
Great, I will try to do that myself if I need help I will ask for help 🙂